app/scripts/migrations/137.test.ts

```diff
@@ -1148,117 +1333,4 @@ describe('migration #137', () => {
       });
     },
   );
-
-  it('skips malformed subjects while migrating valid ones', async () => {
```

Comment:
these still apply, no?

Reply:
it does not. See [here](https://github.com/MetaMask/metamask-extension/pull/29682/files#diff-e069365c347d83015fc87425754a1bde536a4bcdbb0baf52494f216090e1c7d0R264) for example

Reply:
This will be counted as a too f***ed to fix situation